tests/lib/fakestore/cmd/fakestore/cmd_make_refreshable.go

```diff
@@ -24,12 +24,16 @@ import (
 )
 
 type cmdMakeRefreshable struct {
-	TopDir string `long:"dir" description:"Directory to be used by the store to keep and serve snaps, <dir>/asserts is used for assertions"`
+	TopDir     string `long:"dir" description:"Directory to be used by the store to keep and serve snaps, <dir>/asserts is used for assertions"`
+	SnapBlob   string `long:"snap-blob" description:"File or directory with new snap revision contents"`
+	Positional struct {
+		SnapName string
+	} `positional-args:"yes"`
 }
 
 func (x *cmdMakeRefreshable) Execute(args []string) error {
 	// setup fake new revisions of snaps for refresh
-	return refresh.MakeFakeRefreshForSnaps(args, x.TopDir)
+	return refresh.MakeFakeRefreshForSnaps(x.Positional.SnapName, x.TopDir, x.SnapBlob)
```

Comment:
So we don't support setting multiple snaps at once anymore and instead the tool has to be called for every snap? That's fine, I guess if we shouldn't error out if there are extra args passed (or will it already error out?)